revenuecat_examples/purchase_tester/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Dec 16 14:24:29 UYT 2020
+#Mon Feb 06 12:56:20 CST 2023
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```

Comment:
Hmm I wonder i we should move some of these changes to `main` and then rebase this... NABD though